build.rs

```diff
@@ -514,18 +514,25 @@ fn main() {
         }
 
         // Check additional required libraries.
+        // Flags are now of the type `EXTRALIBS-avcodec`
         {
             let config_mak = source().join("ffbuild/config.mak");
             let file = File::open(config_mak).unwrap();
             let reader = BufReader::new(file);
             let extra_libs = reader
                 .lines()
-                .find(|ref line| line.as_ref().unwrap().starts_with("EXTRALIBS"))
+                .filter(|ref line| line.as_ref().unwrap().starts_with("EXTRALIBS"))
```

Comment:
We can improve it by just using `filter_map` and avoid all filter/map/ref combo. It will be something like
```
.filter_map(|line| line.map(|l| l.starts_with("EXTRALIBS")).ok())
```

Reply:
After asking around #rust, it seems what I did was the cleanest - you can't use `flat_map` directly since `starts_with` returns a `boolean` and `lines` a `Result`